deployer/scripts/install.sh

```diff
@@ -255,7 +255,13 @@ function generate_configmaps() {
     # generate curator configmap
     oc create configmap logging-curator \
       --from-file=config.yaml=conf/curator.yml
-    oc label configmap/logging-curator logging-infra=support # make easier to delete later
+    oc label configmap/logging-curator logging-infra=support
+
+    # generate fluentd configmap
+    oc create configmap logging-fluentd \
+      --from-file=fluent.conf=conf/fluent.conf \
+      --from-file=throttle-config.yaml=conf/fluentd-throttle-config.yaml || return
```

Comment:
My primary concern is that right now there are 4 different places where fluentd configuration is defined:
1) hardcoded config files
2) dynamically generated config files
3) config files passed via configmaps
4) env variables

I think this is not an easy setup.
User needs to have knowledge of fluentd, then user also needs to understand 1)-4) to know where to add his modifications(because w/o knowing the full config and where it is coming from it is very easy to screw up).
On update there are 4 different items to track.

To me personally it is much easier to see the entire fluentd configuration accurately(i.e. dynamically generated configs output to configmap, rather than to .conf file) taken from configmaps.
I'm not arguing against good directory structure which will advise how/where users need to add their modifications.
If everything(most stuff) is in configmaps on upgrade you can still check what was edited and act accordingly.